includes/Media/Media_Source.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Class Media_Source
+ *
+ * @link      https://github.com/googleforcreators/web-stories-wp
+ *
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ */
+
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare(strict_types = 1);
+
+namespace Google\Web_Stories\Media;
+
+use ReflectionClass;
+
+/**
+ * Media source "enum" holding all possible term names for the media source taxonomy.
+ */
+class Media_Source {
```

Comment:
Why is this it's own class? Could it not just be part of `Media_Source_Taxonomy`?

Reply:
So we can easily convert it to a [proper enum](https://php.watch/versions/8.1/enums) once our PHP version requirement increases to 8.1.

Reply:
Okay, but that being a min requirement seems to be years off. Many years off. It is only currently 3.9% of WordPress sites and is only just officially supported by WP core. As yourself like to remind me, don't write for a future. Once that case in 2-3 years, then refactor then. For now, leave a comment in the code for future devs to work it out.

Reply:
Still believe these should be one class.